plugins/kubernetes/app/controllers/kubernetes/stages_controller.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class Kubernetes::StagesController < ResourceController
+  include CurrentProject
+  include CurrentStage
+
+  DEFAULT_BRANCH = "master"
```

Comment:
bet we got that somewhere already ... reuse would be nice

Reply:
yep but they seem all to be local to their specific controller. not sure where to put it